Backpack/Switch/Classes/BPKSwitch.m

```diff
@@ -33,8 +33,6 @@ - (instancetype)initWithCoder:(NSCoder *)coder {
     BPKAssertMainThread();
     self = [super initWithCoder:coder];
     if (self) {
-        _primaryColor = BPKColor.skyBlue;
```

Comment:
This is a nullable property (for theming), and if it's not set we just use skyBlue. So there was no point in setting this.